modules/openy_traction_rec_import/src/Commands/OpenyTractionRecImportCommands.php

```diff
@@ -199,11 +181,17 @@ public function cleanUp(array $options) {
    */
   public function fetch() {
     if (!$this->tractionRecFetcher->isEnabled()) {
-      $this->logger()->notice(dt('Fetcher is disabled!'));
+      $this->logger()->notice('Fetcher is disabled!');
```

Comment:
We need to be more specific, because usually there are multiple diferent fetchers in a system
```suggestion
      $this->logger()->notice('Traction REC: Fetcher is disabled!');
```